charts/generic-service/values.schema.json

```diff
@@ -937,6 +937,27 @@
                 "additionalProperties": false
               },
               "description": "Alerts for messages of specific log levels"
+            },
+            "lokiSideCar": {
+              "type": "object",
+              "properties": {
+                "configMapLabel": {
+                  "type": "string",
+                  "default": "loki_rules",
```

Comment:
This value is unset by default, right?

Reply:
That's the default in the loki helm chart. But it can be set to another value. So I think I should set `configMapLabel ` and `configMapLabelValue ` in `values.yaml` as well.

Reply:
I have added the loki sidecar variables to values.yaml, so the ConfigMap will be created unless the user sets `alerting.logs.lokiSideCar` to a falsy value explicitly.